console/src/components/Services/Data/RawSQL/ResultTable.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useMemo } from 'react';
+import ReactTable from 'react-table';
+import styles from '../../../Common/TableCommon/Table.scss';
+
+interface ResultTableProps {
+  headers: string[];
+  rows: Array<string[]>;
+}
+const getTableConfig = (headers: string[], rows: Array<string[]>) => {
+  const columns = headers.map(i => ({
+    Header: i,
+    accessor: i,
+  }));
+  const data = rows.map(i => {
+    const result: Record<any, string> = {};
```

Comment:
Would this work instead?
```suggestion
    const result: Record<string, string> = {};
```

Reply:
done